### ISSUE TITLE
Porting to Qt5

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#256]
```
Port the current code to Qt5 keeping code compatibility to Qt 4.8x. 
```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 22 Dec 2012 at 3:55

**Comment by GoogleCodeExporter** [comment]
```
Are actually ported to Qt5:
- FreeMedForms EMR
- FreeDiams
- FreeToolBox
```

Original comment by `eric.mae...@gmail.com` on 23 Dec 2012 at 12:43

**Comment by GoogleCodeExporter** [comment]
```
- FreePad (with a segfaulting -> see FIXME)
```

Original comment by `eric.mae...@gmail.com` on 23 Dec 2012 at 12:55

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 7ae4209c30c7.
```

Original comment by `eric.mae...@gmail.com` on 23 Dec 2012 at 1:01
- Changed state: **Fixed**
